src/components/ImageView/index.native.js

```diff
@@ -107,44 +82,38 @@ class ImageView extends PureComponent {
         return false;
     }
 
-    imageLoadingStart() {
-        this.setState({isLoading: true});
+    imageLoad({nativeEvent}) {
+        // Wait till animations are over to prevent stutter in navigation animation
+        this.state.interactionPromise = InteractionManager.runAfterInteractions(() => {
```

Comment:
I wonder if we should just create a class variable instead of attaching this to state? Any reason one way is better than the other?

Reply:
I think that state was there before this PR, but it should be reactive to changes such as [here](https://github.com/margelo/expensify-app-fork/blob/f97273b6700252e1431250d7c7a71f91e5d65fe9/src/components/ImageView/index.native.js#L178) where its used to keep the image hidden while layout calculations are performed

Reply:
Aah ok cool sounds good to me! Let's keep it :D